src/main/java/core/basesyntax/service/FruitConverter.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+
+public class FruitConverter {
+    public static Fruit convertToFruit(String name) {
+        for (Fruit fruit: Storage.keyset()) {
+            if (fruit.getName().equals(name)) {
+                return fruit;
+            }
+            break;

```

Comment:
There is a 'break' statement that will cause the loop to exit after the first iteration. It should be removed to allow the loop to iterate over all elements in the set.

Reply:
ok, done